lib/interviewer/selectors/name-generator.js

```diff
@@ -23,13 +24,27 @@ const stageCardOptions = (_, props) => props.stage.cardOptions;
 const stageSortOptions = (_, props) => props.stage.sortOptions;
 const propPanels = (_, props) => props.stage.panels;
 
+const getIDs = createSelector(
+  propStageId,
+  getPromptId,
+  (stageId, promptId) => {
+    return {
+      stageId,
+      promptId,
+    };
+  },
+)
+
 export const getPromptModelData = createSelector(
   getStageSubject,
-  getIds,
-  ({ type }, ids) => ({
-    type,
-    ...ids,
-  }),
+  getIDs,
+  ({ type }, { stageId, promptId }) => {
+    return {
+      type,
+      stageId,
+      promptId,
+    };
+  },
```

Comment:
This is the actual "fix" (although the real fix is in the new implementation of the getIDs selector, so as not to use props).

I also changed the style a little to remove the spread operator, which is something I want us to do more broadly. Spread operators can make it difficult to debug problems because it isn't clear which data they contain.